jobs/env/pull-kubernetes-e2e.env

```diff
@@ -4,3 +4,6 @@ KUBE_GCS_UPDATE_LATEST=n
 
 # NUM_NODES and GINKGO_PARALLEL_NODES should match kubernetes-e2e-gce.
 NUM_NODES=4
+
+# Enable the PodSecurityPolicy in tests.
+ENABLE_POD_SECURITY_POLICY=true
```

Comment:
I'm unclear what uses this env file... would this enable it for pull-kubernetes-e2e-gce-etcd3?

Reply:
Ah, good point, I thought these env files mapped 1:1 to jobs. I moved this to `pull-kubernetes-e2e-gce-etcd3.env`, which is used by the `pull-kubernetes-e2e-gce-bazel` and `pull-kubernetes-e2e-gce-etcd3` jobs (see https://github.com/kubernetes/test-infra/blob/master/jobs/config.json)

Reply:
is there a reason we wouldn't want this on by default in `gce/config-test.sh`? might require targeted disabling in tests that want to approximate production (e.g. scale tests) but that would give me more confidence it was actually being exercised

Reply:
That's what I originally had. However, so many of our tests fail all the time that I wasn't confident that the PSP wasn't breaking them, and they aren't run on pulls. I could just submit it, and then fix anything that breaks in CI, but I opted for a gradual approach instead. I'm open to changing it...

Reply:
ok. if this is a transitional step, add a TODO here to move it into `gce/config-test.sh`, then LGTM

Reply:
Done.